### ISSUE TITLE
Documentation popup with link opens the content in the same tab

### ISSUE THREAD

**Issue by smalers** [OpenWaterFoundation/owf-app-info-mapper-ng#230]
Clicking on a link in a popup markdown documentation dialog for a layer opens the link page in the same tab.  This used to work OK.  The same behavior may occur for map and group levels.

Also, I had some trouble opening group level documentation markdown.  Ultimately I took those documents out because they did not add much value.  I was getting warnings about the path not found even though they existed.  I don't think it was a typo on my part.

**Comment by Nightsphere** [comment]
The openLinksInNewWindow attribute has now been updated for documentation so a new tab will be opened when a link is clicked. There's supposed to be a global way to set that attribute so it happens project-wide I thought, but apparently I did it wrong, so I fixed it on a more granular level for now. I will talk to Sofia about getting the global setting to work.

As for the group level docs, I'll try to do some more testing with the paths and see what's going on
